examples/06_Combining_Models.ipynb

```diff
@@ -250,6 +250,247 @@
+    "plt.xlabel('Time (s)')\n",
+    "plt.ylabel('Voltage')\n",
+    "plt.legend()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Much better!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The same ensemble approach can be used with a heterogeneous set of models that have different states.\n",
+    "\n",
+    "Here we will repeat the exercise using the battery electrochemisty and equivalent circuit models. The two models share one state in common (tb), but otherwise are different"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from progpy.models import BatteryElectroChemEOD\n",
+    "m_electro = BatteryElectroChemEOD(qMobile=7800)\n",
+    "\n",
+    "print('Electrochem states: ', m_electro.states)\n",
+    "print('Equivalent Circuit States', m_circuit.states)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's create an ensemble model combining these and evaluate it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m_ensemble = EnsembleModel((m_circuit, m_electro), process_noise = 0, measurement_noise=0)\n",
+    "results_ensemble = m_ensemble.simulate_to(t_end, future_loading)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To compare these results, lets also simulate the two models that comprise the ensemble model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results_circuit1 = m_circuit.simulate_to(t_end, future_loading)\n",
+    "results_electro = m_electro.simulate_to(t_end, future_loading)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The results of each of these are plotted below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure()\n",
+    "plt.plot(results_circuit1.times, [z['v'] for z in results_circuit1.outputs], color='blue', label='circuit')\n",
+    "plt.plot(results_electro.times, [z['v'] for z in results_electro.outputs], color='red', label='electro chemistry')\n",
+    "plt.plot(results_ensemble.times, [z['v'] for z in results_ensemble.outputs], color='yellow', label='ensemble')\n",
+    "plt.plot(test_time, data[RUN_ID]['voltage'], color='green', label='ground truth')\n",
+    "plt.legend()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that the result may not be exactly between the other two models. This is because of aggregation is done in 2 steps: at state transition and then at output calculation.\n",
```

Comment:
What do you think about moving this somewhere earlier in the example and adding a bit more detail? Users may be interested in the basics of how aggregation works, and then when we pick 'median' as an aggregator this may make more sense

Reply:
I couldn't find a better spot for it. Do you have any suggestions?